src/modules/create-market/helpers/build-create-market.js

```diff
@@ -0,0 +1,51 @@
+import { augur } from 'services/augurjs'
+import speedomatic from 'speedomatic'
+import { BINARY, CATEGORICAL, SCALAR } from 'modules/markets/constants/market-types'
+import { DESIGNATED_REPORTER_SELF } from 'modules/create-market/constants/new-market-constraints'
+
+export const buildCreateMarket = (newMarket, isEstimate, universe, loginAccount, contractAddresses) => {
+  const tags = []
+  if (newMarket.tag1) tags.push(newMarket.tag1)
+  if (newMarket.tag2) tags.push(newMarket.tag2)
+  // General Properties
+  const formattedNewMarket = {
+    universe: universe.id,
+    _endTime: parseInt(newMarket.endDate.timestamp, 10),
+    _feePerEthInWei: speedomatic.fix(newMarket.settlementFee / 100, 'hex'),
+    _denominationToken: contractAddresses.Cash,
+    _description: newMarket.description,
+    _designatedReporterAddress: newMarket.designatedReporterType === DESIGNATED_REPORTER_SELF ? loginAccount.address : newMarket.designatedReporterAddress,
+    _topic: newMarket.category,
+    _extraInfo: {
+      marketType: newMarket.type,
+      longDescription: newMarket.detailsText,
+      resolutionSource: newMarket.expirySource,
+      tags,
+    },
+  }
+
+  if (isEstimate) {
+    formattedNewMarket.tx = { estimateGas: true, gas: augur.constants.DEFAULT_MAX_GAS }
+  }
+  // Type Specific Properties
```

Comment:
`Properties specific to each market type`?

Reply:
was copied and pasted from existing code, will update to make more informative